### PR TITLE
Add `ampform-dpd` from PyPI

### PR DESCRIPTION
**Pull request by redeboer**
See https://pypi.org/project/ampform-dpd

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Closes https://github.com/ComPWA/ampform-dpd/issues/120

### CODE REVIEW

**Comment by shenvitor** [comment]
Comment testing  :)

File: recipes/ampform-dpd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  imports:
+    - ampform_dpd
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  dev_url: https://github.com/ComPWA/ampform
```
Comment (synapticarbors) [inline recipes/ampform-dpd/meta.yaml:39]: shouldn't the `dev_url` be the same as the home url?

File: recipes/ampform-dpd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "ampform-dpd" %}
+{% set version = "0.1.0" %}
```
Comment (synapticarbors) [inline recipes/ampform-dpd/meta.yaml:2]: Is there a reason to be starting with this old version that looks like it was released in Dec 2022?

Comment (synapticarbors) [reply recipes/ampform-dpd/meta.yaml:2]: That's reasonable if that's what you want to do, although you're not required to.

**Pull request merged**